lib/Doctrine/Annotations/Metadata/Reflection/ClassReflectionProvider.php

```diff
@@ -4,13 +4,9 @@
 
 namespace Doctrine\Annotations\Metadata\Reflection;
 
-use Doctrine\Annotations\Metadata\Reflection\Exception\InvalidClass;
 use ReflectionClass;
 
 interface ClassReflectionProvider
 {
-    /**
-     * @throws InvalidClass
```

Comment:
should stay

Reply:
AFAIR I removed it (class) because it was not used and referred to non existent classes

Reply:
maybe not yet implemented :)